DXMainClient/Domain/Multiplayer/CnCNet/MapSharer.cs

```diff
@@ -370,7 +370,7 @@ private static void Download(object details)
         }
 
         public static string GetMapFileName(string sha1, string mapName)
-            => mapName.ToWin32FileName() + "_" + sha1;
+            => $"{mapName.ToWin32FileName()}_{sha1}.{ClientConfiguration.Instance.MapFileExtension}";
```

Comment:
There should be no extensions here

Reply:
who even thought having a PARTIAL file name and calling it just a file name is a good idea...

Reply:
blame https://github.com/CnCNet/xna-cncnet-client/pull/243 :)